sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/authentication/RefreshTokensImpl.java

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Is this class handwritten? The license header doesn't have this line `// Code generated by Microsoft (R) AutoRest Code Generator.` which is generally included for all generated files.

Reply:
Yes, as mentioned in another file, a bug tracking this already exists.

---
In reply to: [597960498](https://github.com/Azure/azure-sdk-for-java/pull/19926#discussion_r597960498) [](ancestors = 597960498)